store/sqlstore/emoji_store.go

```diff
@@ -105,13 +105,13 @@ func (es SqlEmojiStore) GetByName(name string) (*model.Emoji, *model.AppError) {
 	var emoji *model.Emoji
 
 	if err := es.GetReplica().SelectOne(&emoji,
-		`SELECT
-				*
-			FROM
-				Emoji
-			WHERE
-				Name = :Name
-				AND DeleteAt = 0`, map[string]interface{}{"Name": name}); err != nil {
+	`SELECT
```

Comment:
This exact line was correctly placed, was the other lines of the string that were shifted.